content/docs/guides/upgrade.md

```diff
@@ -72,7 +72,7 @@ $ osm mesh upgrade
 OSM successfully upgraded mesh osm
 ```
 
-This command will upgrade the mesh with the default mesh name in the default OSM namespace. Values from the previous release will carry over to the new release except for `osm.image.registry` and `osm.image.tag` which are overridden by default. For example, if OSM v0.7.0 is installed, `osm mesh upgrade` for {{< param osm_version >}} of the CLI will update the control plane images to {{< param osm_version >}} by default.
+This command will upgrade the mesh with the default mesh name in the default OSM namespace. Values from the previous release will NOT carry over to the new release by default, but may be passed with the `--set` flag on `osm mesh upgrade`.
```

Comment:
Do you think it's worth giving an example? Upon reading this, I find myself wondering exactly how the `--set` flag works in this case, like, do I need to set specific variables with explicit values? Or if I say `--set` then will it pull them all in? etc.

Reply:
Does the `--help` text for that flag clarify? https://github.com/openservicemesh/osm/blob/903f36cde7ae2848c045ef68f7e5018f5dc8ff0e/cmd/cli/mesh_upgrade.go#L87
> Set arbitrary chart values (can specify multiple or separate values with commas: key1=val1,key2=val2)

Reply:
Yes, that's more detail - I think that the docs are a good place to surface that, since people may have the question before they are in a position to try various flags.

Reply:
I think I'll leave notes on the specific syntax for that flag in the `--help` text, but I've reworded this to make the overall behavior clearer.